src/site/content/en/metrics/inp/index.md

```diff
@@ -252,6 +259,40 @@ See the [`onINP()`](https://github.com/GoogleChrome/web-vitals/tree/next#oninp)
 Gathering INP metrics in the field will only work on browsers that [fully support the Event Timing API](https://caniuse.com/mdn-api_performanceeventtiming), including its `interactionId` property.
 {% endAside %}
 
+You can also measure INP in browsers that support the Event Timing API without a library using vanilla JavaScript available in the expandable section below.
```

Comment:
This snippet doesn't actually measure INP. I think our message with this section/snippet should be something like:
- Measuring INP in JS is tricky, so we recommend the web-vitals lib for  most accurate results
- But to get a quick sense of what the longest interactions look like on your page, you can run this code (and modify it to suit your needs).